test/tests/functions/baselinecorrection/tests

```diff
@@ -0,0 +1,77 @@
+[Tests]
+  issues = '#296'
+  design = functions/BaselineCorrection.md
+
+  [test_correction]
+    requirement = "The BaselineCorrection class shall accurately adjust a raw acceleration time history according to the method of using least squares polynomial fits."
+
+    [half_sine_scaled]
+      type = CSVDiff
+      input = half_sine_scaled.i
+      csvdiff = half_sine_scaled_out.csv
+
+      detail = "The output should be a displacement time history scaled by a factor of 4.886 that does not exhibit drifting behavior. The input acceleration time history is a half cycle of `9.87 \sin(\pi t)`."
+    []
+    [twenty_sines_with_bc]
+      type = CSVDiff
+      input = twenty_sines_with_bc.i
+      csvdiff = twenty_sines_with_bc_out.csv
+
+      detail = "The output should be an adjusted acceleration time history that is compatible with ICs and BCs objects and the displacement at the mesh boundary where it is prescribed should not exhibit drifting behavior. The input acceleration time history is twenty cycles of `-250 \pi^{2} \sin(50 \pi t)`."
+    []
+  []
+
+  [test_errors]
+    requirement = "The BaselineCorrection class shall appropriately invoke `mooseError()` when the input configuration is not supported."
+
+    [no_input]
+      type = RunException
+      input = no_input_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: Either `data_file` or `time_values` and `acceleration_values` must be specified exclusively."
+
+      detail = "This error shall occur since no acceleration time history was provided."
+    []
+    [extra_input]
+      type = RunException
+      input = extra_input_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: Either `data_file` or `time_values` and `acceleration_values` must be specified exclusively."
+
+      detail = "This error shall occur since the parameters which control the different input types were not provided exclusively."
+    []
+    [missing_value_param]
+      type = RunException
+      input = missing_value_param_error.i
+      expect_err = "In BaselineCorrection corrected_accel_func: Either `data_file` or `time_values` and `acceleration_values` must be specified exclusively."
+
+      detail = "This error shall occur since the `time_values` and `acceleration_values` parameters were not provided together."
```

Comment:
the `time_values` and `acceleration_values` parameters are not provided together.